### PR TITLE
Unify creation of testing ChainSpec

### PR DESCRIPTION
**Pull request by nearmax**
TODO in future PR: For prod DevNet and TestNet we should not have built-in ChainSpecs, and instead they should always be generated with proper random seed. Build-in ChainSpecs should be only used inside tests.

### CODE REVIEW

File: node/configs/src/chain_spec.rs [hunk 1]
```diff
@@ -58,70 +68,105 @@ impl ChainSpec {
...
+    CAROL_ID,
+    "dan.near",
+    "eve.near",
+    "frank.near",
+    "grace.near",
+    "heidi.near",
+    "ivan.near",
+    "judy.near",
+    "mike.near",
+    "niaj.near",
```
Comment (bowenwang1996) [inline node/configs/src/chain_spec.rs]: This feels like a strange name.

File: node/configs/src/chain_spec.rs [hunk 2]
```diff
@@ -58,70 +68,105 @@ impl ChainSpec {
...
+    "olivia.near",
+    "pat.near",
+    "sybil.near",
+    "trudy.near",
+    "victor.near",
+    "wendy.near",
+];
+
+/// Type of id to use for the default ChainSpec. "alice.near" is a named id, "near.0" is an
```
Comment (bowenwang1996) [inline node/configs/src/chain_spec.rs]: what is "near.0"

File: node/runtime/src/system.rs [hunk 3]
```diff
@@ -384,7 +388,8 @@ mod tests {
     fn test_refund_on_send_money_to_non_existent_account() {
         let (runtime, trie, root) = get_runtime_and_trie();
         let (mut alice, root) = User::new(runtime, &alice_account(), trie.clone(), root);
-        let (new_root, apply_results) = alice.send_money(root, &eve_account(), 10);
+        let money_used = 10;
```
Comment (bowenwang1996) [inline node/runtime/src/system.rs]: It seems that `money_used` is not necessary here.

File: node/configs/src/chain_spec.rs (hunk 1 above)
Comment (nearmax) [reply node/configs/src/chain_spec.rs]: Standard name used in cryptography.

File: node/configs/src/chain_spec.rs (hunk 2 above)
Comment (nearmax) [reply node/configs/src/chain_spec.rs]: "near.0" is an enumerated id.

File: node/runtime/src/system.rs (hunk 3 above)
Comment (nearmax) [reply node/runtime/src/system.rs]: Done.

**Comment by nearmax** [comment]
@bowenwang1996 PTAL

**Pull request merged**